ext/standard/user_filters.c

```diff
@@ -384,38 +386,42 @@ PHP_FUNCTION(stream_bucket_make_writeable)
 static void php_stream_bucket_attach(int append, INTERNAL_FUNCTION_PARAMETERS)
 {
 	zval *zbrigade, *zobject;
-	zval *pzbucket, *pzdata;
+	zval *pzbucket, *pzdata, rv;
 	php_stream_bucket_brigade *brigade;
 	php_stream_bucket *bucket;
 
 	ZEND_PARSE_PARAMETERS_START(2, 2)
 		Z_PARAM_RESOURCE(zbrigade)
-		Z_PARAM_OBJECT(zobject)
+		Z_PARAM_OBJECT_OF_CLASS(zobject, stream_bucket_class_entry)
 	ZEND_PARSE_PARAMETERS_END();
 
-	if (NULL == (pzbucket = zend_hash_str_find_deref(Z_OBJPROP_P(zobject), "bucket", sizeof("bucket")-1))) {
-		zend_argument_value_error(2, "must be an object that has a \"bucket\" property");
+	if ((brigade = (php_stream_bucket_brigade*)zend_fetch_resource(
+		Z_RES_P(zbrigade), PHP_STREAM_BRIGADE_RES_NAME, le_bucket_brigade)) == NULL) {
```

Comment:
Are there any coding rules for adding a line break in the middle of if?

Personally, in the current state, I can't discern from the indentation whether it's an "if conditional statement" or "the processing content of an if block", so I think it would be easier to understand if the indentation were one step deeper.

Reply:
I don't think we have a CS for this, the formatting of code is very inconsistent and IIRC last time someone tried to use ``clang-format`` there were some issues?

Reply:
I see, thx.

Unless there is a specific rule, IMHO I think it's better to use deeper indentation. (I think readability varies from person to person, so I don't intend to force it.)

Reply:
TBH I'm on the other side as I don't like deeper indentation, it looks weird to me 😅